### ISSUE TITLE
Closure does not have void return type hint.

### ISSUE THREAD

**Issue by Jurigag** [slevomat/coding-standard#887]
Hi, i have a question, why exactly rule `SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint` with `Closure does not have void return type hint` is triggered in code like this:

```
  /**
     * @return void
     */
    public function handleCampaignBeforeDeleted(): void
    {
        $this->eventManager->attach($this->getEventName(), function (
            $event,
            CampaignBeforeDeletedEvent $source
        ) {
            // empty :)
        });
    }
```

Not sure what i can do here, other than moving closure to method or something like this.

**Comment by kukulich** [comment]
You should add `void` return type hint to the closure :)

**Comment by Jurigag** [comment]
Any example how to do it?
